### DISCUSSION TITLE
Watch upgrade progress

### DISCUSSION THREAD

**Discussion by ktreptow** [concourse/concourse#7682] [Help & Support]
Hi Concourse-Team! 
I want to upgrade a Concourse instance in EC2 instances to the newest version and was wondering, if there is any way to check the progress of the upgrade. Especially the database migration in version [v7.0.0](https://github.com/concourse/concourse/releases/tag/v7.0.0)

Thanks!

**Comment by taylorsilva** [comment]
No fancy progress bar anywhere sadly. I would recommend trying to use the `concourse migrate` command if you want to do the migration separately. When that exits then you'll know the db migration is done.
